libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -179,3 +179,8 @@ singleton AP_Baro alias baro
 singleton AP_Baro method get_pressure float
 singleton AP_Baro method get_temperature float
 singleton AP_Baro method get_external_temperature float
+
+include AP_ESC_Telem/AP_ESC_Telem.h
+singleton AP_ESC_Telem alias esc_telem
+singleton AP_ESC_Telem method tcan_is_active boolean
+singleton AP_ESC_Telem method get_usage boolean uint8_t 0 12 uint32_t'Null
```

Comment:
Can we find a define for the upper limit of 12?

Reply:
Yes, I've changed it to use the NUM_SERVO_CHANNELS define from the SRV_Channel library (see here: https://github.com/ArduPilot/ardupilot/blob/master/libraries/SRV_Channel/SRV_Channel.h#L24)